exonum/src/sandbox/consensus/round_details.rs

```diff
@@ -1335,8 +1335,8 @@ fn handle_precommit_positive_scenario_commit_with_queued_precommit() {
     let sandbox = timestamping_sandbox();
     let sandbox_state = SandboxState::new();
 
-    let block_1_delay = 2 * sandbox.round_timeout() + PROPOSE_TIMEOUT + 1;
-    let block_2_delay = 2 * sandbox.round_timeout() + 2 * PROPOSE_TIMEOUT + 1;
+    let block_1_delay = 2 * sandbox.current_round_timeout() + PROPOSE_TIMEOUT + 1;
+    let block_2_delay = 2 * sandbox.current_round_timeout() + 2 * PROPOSE_TIMEOUT + 1;
```

Comment:
I think if round timeout is not fixed then we can't do `2 * sandbox.current_round_timeout()` anymore and have to be more elaborate. It may work with current configuration values, but may suddenly break if someone tweaks them.

Reply:
I thought about it, but this parameter is not used by block builder anyway, should be revisited.